stat/distuv/uniform_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math/rand"
```

Comment:
Could you make this "golang.org/x/exp/rand" please?

Reply:
Is this a direct replacement for math/rand? Does this mean that `uniform.go` needs to be changed, as well?